openedx_events/event_bus/avro/tests/test_avro.py

```diff
@@ -97,7 +98,7 @@ def generate_test_event_data_for_data_type(data_type):  # pragma: no cover
     defaults_per_type = {
         int: 1,
         bool: True,
-        str: "default",
+        str: "passing",
```

Comment:
[curious] Why is this the new default?

Reply:
Since this affects all tests, I don't think we should change the default to this specific value for tests to pass. However, we should be able to add validators like this without breaking all tests. I'll look into it.

Reply:
If you prefer, you could split this PR into two to include the event definition in master and then the event bus support so both things can move separately.

Reply:
We can move forward without it then! Still, I'll let you know what I find.